src/components/TaxonomyTerms/TaxonomyTerms.js

```diff
@@ -0,0 +1,16 @@
+import Link from 'next/link';
+import styles from './TaxonomyTerms.module.scss';
+
+export default function TaxonomyTerms({ post, taxonomy }) {
+  const termLinks = post?.[taxonomy]()?.edges.map((edge) => {
+    const { name, uri } = edge?.node;
+    return uri && <Link href={uri}>{name}</Link>;
+  });
+
+  return (
```

Comment:
Could be nice to return `null` if `termLinks` is empty so you don't get empty lists.
<img width="331" alt="Screen Shot 2022-03-21 at 3 21 24 PM" src="https://user-images.githubusercontent.com/4661832/159348593-ad7188c8-4613-4432-9b2e-9f9bec42c99e.png">